build.psm1

```diff
@@ -719,9 +730,13 @@ function Start-PSPester {
 
     Write-Verbose "Running pester tests at '$path' with tag '$($Tag -join ''', ''')' and ExcludeTag '$($ExcludeTag -join ''', ''')'" -Verbose
     Publish-PSTestTools
+
```

Comment:
I added import of helper modules in current session too so it is not required to use `Start-PSPester` - it is worked well `.\Out-Host.Tests.ps1`